File: Python/AlphaNumeric.py
#!/usr/bin/env python
# Prints the given letter n times
# Where n is the preceding number to the letter
def main():
	line = raw_input("Enter a string like 1a2b3c: ")
	for x in range(len(line)):
		if (line[x].isdigit()) and (not line[x + 1].isdigit()):
			for y in range(0, int(line[x])):
				print line[x + 1]

main()

File: Java/AlphaNumeric.java
// Prints the given letter n times
// Where n is the preceding number to the letter
public class AlphaNumeric{
	public static void main(String[] args) {
		String line = new java.util.Scanner(System.in).next();
		for (int i = 0; i < line.length(); i++) {
			if (Character.isDigit(line.charAt(i)) && !Character.isDigit(line.charAt(i + 1))){
				for (int j = 0; j < Character.getNumericValue(line.charAt(i)); j++) {
					System.out.print((line.charAt(i + 1)));
				}
			}
			System.out.println();
		}
	}
}
